jdaviz/app.py

```diff
@@ -129,7 +129,7 @@ class Application(VuetifyTemplate, HubListener):
 
     loading = Bool(False).tag(sync=True)
 
-    def __init__(self, configuration=None, *args, **kwargs):
+    def __init__(self, configuration=None, auto_link=True, *args, **kwargs):
```

Comment:
This breaks existing API. By putting `auto_link` before `*args`, `auto_link` will take whatever second argument you pass in. And since this is top-level Application class, we need to be very careful with the signature.

Reply:
I also don't know how I feel about making auto_link compulsory in top-level app and defaulting to True. What does this mean for all the other viz?

Reply:
Should it come after `*args`?

Reply:
Personally I would pop it off `kwargs` before passing kwargs into something else, but perhaps others have better ideas.

Reply:
Ah, that does sound better.